staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -271,6 +274,11 @@ func InternalError(field *Path, err error) *Error {
 	return &Error{ErrorTypeInternal, field.String(), nil, err.Error()}
 }
 
+// ChangeForbidden returns a *Error indicating field change is disallowed*. This is used to signal
+// callers must also include errors if the new value is invalid independent of the current value.
```

Comment:
Is there a stray copy-paste on this line? `This is used to inform callers that their proposed value would be a forbidden mutation` or something like that.

Would be even better if we could note the difference between a field that is immutable and one that is mutable but just not in this particular way.